c10/macros/Macros.h

```diff
@@ -431,11 +431,11 @@ __device__ __attribute__((noinline)) __attribute__((weak)) void __assert_fail(
 #endif
 
 // Portable determination of whether type T is trivially copyable.
-// Warning: __has_trivial_copy for GCC may not always detect the non-POD
+// Warning: __is_trivially_copyable for GCC may not always detect the non-POD
```

Comment:
Why is this changing?

Reply:
I left a comment here, not sure if we still need this warning as __is_trivially_copyable is much 'better'. I updated the comment just in case but I assume we can get rid of this.